src/components/Input/InputWithTooltip/InputWithTooltip.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ComponentProps } from 'react'
+import styled from 'styled-components'
+
+import { Input } from '../Input'
+import { Tooltip as shrTooltip } from '../../Tooltip'
+
+type Props = ComponentProps<typeof Input> & {
+  /** 入力欄に紐付けるツールチップに表示するメッセージ */
+  tooltipMessage: React.ReactNode
+}
+
+export const InputWithTooltip: React.FC<Props> = ({ tooltipMessage, ...props }) => {
+  return (
+    <Tooltip message={tooltipMessage} tabIndex={-1} ariaDescribedbyTarget="inner">
+      <Input {...props} />
```

Comment:
`prefix`のアイコンの`alt` と `<input />` が結びついてないので、`<label>` でラップすると良さそう。

```suggestion
      <label>
        <Input {...props} />
      </label>
```

Reply:
`label` で囲うと用途が限定されすぎるかも、と思ったんですが、絞っちゃってもよさそうですね。